exercises/book/P-final-project/mod.md

```diff
@@ -0,0 +1,58 @@
+# Module P - Final project
+
+It is time to submit a proposal for the final project.
+
+- Form groups of 2 or 3 people. Working solo is not permitted.
+- Build a small rust project
+- Up to 2 groups can work on the same topic
+
+The proposal needs to be submitted by _30th of March, 2023_ by opening a Github repo, placing your proposal there, and granting access to [`@hdoordt`](https://github.com/hdoordt/).
+
+The proposal must contain the following sections:
+- Your names
+- Introduction to your idea
+- Requirements in brief
```

Comment:
Tip: Beware to check that students know what this means (i.e. have they had a course on software engineering? I know this seems like a silly thing to ask but academic programs don't focus on this.)

Reply:
```suggestion
- Requirements in brief (half a page of text at maximum)
```

Reply:
Also, probably good to specify what 'brief' means; the suggestion of course can be changed to "at most a page" or something. But I assume the objective is that students think about their project, not produce a detailed requirements document (nor do we have the time to read many of those)

Reply:
> But I assume the objective is that students think about their project, not produce a detailed requirements document

Exactly!